ui/src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import App from './containers/App';
```

Comment:
Why is App in containers? Seems like most React apps put in either components/ or src/.

Reply:
I'm dividing app structure here into Presentational/Container components as referenced [here](https://daveceddia.com/react-project-structure/). I think this is a good design for our needs, which are going to be a pretty even split between pure UI and custom logic. App.js is a logical container for stuff, and would go under containers.

How we should structure the app is open for discussion, although it's pretty trivial to change in the future and the app is small enough for now that it doesn't really matter.

Reply:
I did some investigation. I think the containers vs components distinction is mainly useful for Redux.

Out of the 12 sample apps at https://reactjs.org/community/examples.html, 3 use containers/. All 3 use redux. Out of the 9 that don't, 1 uses redux.

I find containers vs components confusing. A Facet is a "logical container for stuff"; why isn't that a container?

https://daveceddia.com/react-project-structure/ says "Containers are the stateful ones, and the ones that make the API calls.". But then that requires someone to know which React components make API calls. Component X could make an API call today, but if we rearranged the UI, X could not make an API call tomorrow.

Something about Redux forces people to make the distinction between "components-that-make-API-calls" and "component-don't-make-API-calls". Since we're not using Redux, I'd rather not make people think about that.

As a side note -- while things like this are easy to change in the future, I prefer to have the (short) discussion and change now, rather than after code is submitted.

Reply:
Okay, I moved `App.js` up to the `src/` directory and all of our components into `component/`. As we scale up, we'll organize components by feature hierarchy.